.templates/go/default.mk

```diff
@@ -48,6 +51,9 @@ update-dependencies:
 	cd $(MOD_DIR) && go get -u && go mod tidy
 .PHONY: update-dependencies
 
+pre-release: update-major update-dependencies clean default
```

Comment:
should be:

```makefile
pre-release: remove-replaces update-major update-dependencies clean default
```

Just saw that, I finish the release of another package and fix that.